src/lib/components/submit/SubmitAddNodeProvider.svelte

```diff
@@ -49,38 +69,77 @@
 	const debounceSave = debounce(save);
 
 	$: nodeProviderId,
-		title,
-		summary,
+		nodeProviderName,
+		url,
+		urlSelfDeclaration,
+		urlIdentityProof,
+		hashSelfDeclaration,
+		hashIdentityProof,
 		(() => {
-			if (nodeProviderId === '' && title === '' && summary === '') {
+			if (
+				nodeProviderId === '' &&
+				nodeProviderName === '' &&
+				url === '' &&
+				urlSelfDeclaration === '' &&
+				urlIdentityProof === '' &&
+				hashSelfDeclaration === '' &&
+				hashIdentityProof === ''
+			) {
 				return;
 			}
 
 			debounceSave();
 		})();
-
-	// TODO: summary should probably be a textarea and review placeholder
 </script>
 
 <h1 class="mb-12 text-4xl font-bold capitalize md:text-6xl">
 	Register Your Node Provider Principal
 </h1>
 
 <h2 class="mb-6 text-2xl">
-	To submit a proposal for adding a new node provider, you need to provide a name, an announcement,
-	and a principal ID.
+	To submit a proposal for adding a new node provider, you need to provide a name, a principal ID,
```

Comment:
Maybe let's shorten the sentence, something like: "To submit a proposal for adding a new node provider, you need to provide a name, various urls and other information". That will be a bit more readable.